src/components/RemoveModal/RemoveModal.scss

```diff
@@ -0,0 +1,72 @@
+.container-modal {
+    position: relative;
+}
+
+.container-modal__block {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    width: 100%;
+    max-width: 400px;
+    height: 300px;
+    background-color: black;
+    color: white;
+    border-radius: 10px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    align-items: center;
+    &:focus {
+        outline: none;
+    }
+    &--close {
+        width: 80%;
+        display: flex;
+        justify-content: flex-end;
+        &-img {
+            width: 30px;
+            cursor: pointer;
+        }
+    }
+    &--title {
+        font-size: 30px;
+    }
+    &--btns {
+        width: 100%;
+        display: flex;
+        justify-content: space-evenly;
+    }
+    &--btn {
+        font-family: 'Amatic SC', cursive;
+        font-weight: 700;
+        font-size: 25px;
+        width: 100px;
+        height: 40px;
+        border-radius: 10px;
+        border: none;
+        cursor: pointer;
+        &-dark {
+            background-color: black;
+            color: white;
+            &:focus {
+                outline: none;
+            }
+            &:hover {
+                background-color: rgb(128, 128, 128);
+            }
+        }
+        &-light {
+            @extend .container-modal__block--btn-dark;
+            background-color: white;
+            color: black;
+        }
+    }
+}
+
+.container-modal__block--dark {
```

Comment:
here you use "--dark" as modifier, and in code above as element. This is inconsistent